packages/pyolite-kernel/src/worker.ts

```diff
@@ -171,6 +281,18 @@ self.onmessage = async (event: MessageEvent): Promise<void> => {
     case 'complete-request':
       return complete(messageContent);
 
+    case 'comm-info-request':
+      return commInfo(messageContent);
+
+    case 'comm-open':
+      return commOpen(messageContent);
+
+    case 'comm-msg':
+      return commMsg(messageContent);
+
+    case 'comm-close':
+      return commClose(messageContent);
+
     default:
```

Comment:
Maybe we can get clever here in type-land, and use the [`switch ... never` pattern](https://javascript.plainenglish.io/never-miss-a-switch-case-with-typescript-684bf5d0e1d1?gi=6215774607fe). Indeed, it might be that it's actually `IKernel<T, U>` where `T` and `U` are the `messageTypes` it accepts/creates respectively.

Reply:
I actually failed using more types in this file, loading type definitions from jupyter lab made webpack add a `export = {} `, at the end of the file, turning the worker blob unusable.

I didn't look deeply into the issue, but yeah I totally agree

Reply:
well, if we explore `comlink`, some of these problems _should_ go away (though i _have_ had some issues with webpack5 vs comlink vs wasm and don't have a resolution yet...)